contracts/sol6/Dao/EmergencyFeeHandler.sol

```diff
@@ -0,0 +1,201 @@
+pragma solidity 0.6.6;
+
+import "../IKyberFeeHandler.sol";
+import "../utils/PermissionGroupsNoModifiers.sol";
+import "../utils/zeppelin/ReentrancyGuard.sol";
+import "../utils/zeppelin/SafeMath.sol";
+import "../utils/Utils5.sol";
+
+contract EmergencyKyberFeeHandler is IKyberFeeHandler, PermissionGroupsNoModifiers, ReentrancyGuard, Utils5 {
+    using SafeMath for uint256;
+
+    uint16 public immutable rewardBps;
+    uint16 public immutable rebateBps;
+    address public kyberNetwork;
+
+    mapping(address => uint256) public feePerPlatformWallet;
+    uint256 public totalPlatformFeeWei; // total balance in the contract that is for platform fee
+    mapping(address => uint256) public rebatePerWallet;
+    uint256 public totalRewardWei;
+
+    event HandleFeeFailed(address[] rebateWallets, uint256[] rebateBpsPerWallet, uint256 feeBRRWei);
+
+    event FeeDistributed(
```

Comment:
can call it feeDistribution
since its not really distributed here.
only logged.